bundles/org.openhab.core.voice/src/main/java/org/openhab/core/voice/internal/cache/TTSLRUCacheImpl.java

```diff
@@ -66,6 +67,11 @@ public class TTSLRUCacheImpl implements TTSCache {
      * current request is not known and may or may not exceed the limit.
      */
     protected long cacheSizeTTS = DEFAULT_CACHE_SIZE_TTS * 1024;
+    /**
+     * Length text limit. If exceeded, the cache will passthrough the request without storing it.
+     * (One can safely assume that long TTS are generated for report, probably not meant to be repeated)
+     */
+    private long maxTextLengthCacheTTS = 150;
```

Comment:
```suggestion
    private long maxTextLengthCacheTTS = DEFAULT_MAX_TEXT_LENGTH_CACHE_TTS;
```

and use that also below